src/GovUk.Education.ExploreEducationStatistics.Notifier/ReleaseNotifier.cs

```diff
@@ -0,0 +1,88 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Model;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Services;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Table;
+using static GovUk.Education.ExploreEducationStatistics.Notifier.Model.NotifierQueues;
+using static GovUk.Education.ExploreEducationStatistics.Notifier.Utils.NotifierUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Notifier
+{
+    // ReSharper disable once UnusedType.Global
+    public class ReleaseNotifier
+    {
+        private readonly ITokenService _tokenService;
+        private readonly IEmailService _emailService;
+        private readonly IStorageTableService _storageTableService;
+
+        public ReleaseNotifier(ITokenService tokenService, IEmailService emailService,
+            IStorageTableService storageTableService)
+        {
+            _tokenService = tokenService ?? throw new ArgumentNullException(nameof(tokenService));
+            _emailService = emailService ?? throw new ArgumentNullException(nameof(emailService));
+            _storageTableService = storageTableService ?? throw new ArgumentNullException(nameof(storageTableService));
+        }
+
+        [FunctionName("ReleaseNotifier")]
+        // ReSharper disable once UnusedMember.Global
+        public void ReleaseNotifierFunc(
+            [QueueTrigger(ReleaseNotificationQueue)] PublicationNotificationMessage notificationMessage,
+            ILogger logger,
+            ExecutionContext context)
+        {
+            logger.LogInformation("{FunctionName} triggered",
+                context.FunctionName);
+
+            var config = LoadAppSettings(context);
+
+            var emailTemplateId = notificationMessage.Amendment
+                ? config.GetValue<string>(AmendedReleasePublicationEmailTemplateIdName)
+                : config.GetValue<string>(OriginalReleasePublicationEmailTemplateIdName);
+
+            var baseUrl = config.GetValue<string>(BaseUrlName);
+            var webApplicationBaseUrl = config.GetValue<string>(WebApplicationBaseUrlName);
```

Comment:
We can't be certain this URL comes with a trailing slash, so it might be a good idea to use the `AppendTrailingSlash` extension method on the string.

Reply:
I've added it here and in existing instances in `PublicationNotifier.cs`. 

`AppendTrailingSlash` exists in `Common`, which wasn't added to `Notifier`, so I've added it.